interfaces/Portal/src/app/services/programs-service-api.service.ts

```diff
@@ -616,6 +627,10 @@ export class ProgramsServiceApiService {
     if (statuses) {
       params = params.append('filter.status', `$in:${statuses.join(',')}`);
```

Comment:
All these query-parameters should maybe be defined in an enum somwehere...
And the `$in:`-part should then also be used as the enum-value, instead of hard-coded like this...

These parameter-values are  "something of the API that the Portal isn't in charge of"...